pkgs/development/tools/build-managers/moon/default.nix

```diff
@@ -0,0 +1,29 @@
+{ lib, rustPlatform, fetchFromGitHub, Security, SystemConfiguration, stdenv }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "moon";
+  version = "v1.5.1";
+
+  src = fetchFromGitHub {
+    owner = "moonrepo";
+    repo = pname;
+    rev = version;
+    hash = "sha256-TA27e0W0XSOC326lnO/mSlJNLGn6roJhd1CrQadWb/U=";
+  };
+
+  cargoHash = "sha256-Q044nxI6VGGal9I31VuZeGLho4KIz/Rzg4Lrn1prj4Y=";
+
+  RUSTFLAGS = "-C strip=symbols";
+
+  buildInputs = lib.optionals stdenv.isDarwin [ Security SystemConfiguration ];
```

Comment:
I think the build failing is due to openssl. Looking at my old derivation it might solved with merging the following (and adding the deps).

```nix
  buildInputs = [ openssl ];
  nativeBuildInputs = [ pkg-config ];

  OPENSSL_NO_VENDOR = 1;
```

Reply:
Thanks !